Dockerfile.dev

```diff
@@ -1,10 +1,10 @@
 FROM node:12
 
 EXPOSE 3001 8082
-COPY ["./package.json", "./package-lock.json", "/app/"]
+COPY ["./package.json", "./package-json-watcher.js", "./package-lock.json", "/app/"]
```

Comment:
Please, create the `scripts` folder in the `/src` dir if it doesn't exist. After that push `package-json-watcher.js` to that folder and remove it from the `COPY` line.